uk.co.compucorp.civicrm.hrcomments/xml/schema/CRM/HRComments/Comment.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<table>
+  <base>CRM/HRComments</base>
+  <class>Comment</class>
+  <name>civicrm_hrcomments_comment</name>
+  <comment>Comments</comment>
+  <add>4.4</add>
+  <log>true</log>
+
+  <field>
+    <name>id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>Unique Comment ID</comment>
+    <add>4.4</add>
+  </field>
+  <primaryKey>
+    <name>id</name>
+    <autoincrement>true</autoincrement>
+  </primaryKey>
+
+  <field>
+    <name>entity_name</name>
+    <type>varchar</type>
+    <length>50</length>
+    <required>true</required>
+    <comment>The Entity name associated with the comment</comment>
+    <add>4.4</add>
+  </field>
+
+  <field>
+    <name>entity_id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>The Entity ID associated with the comment</comment>
+    <add>4.4</add>
+  </field>
```

Comment:
We should probably add an index with the entity_name and entity_id fields, since all the queries to get comments will be based on them. You can check the LeaveBalanceChange schema xml to see an example